mlfinlab/portfolio_optimization/hcaa.py

```diff
@@ -41,40 +44,37 @@ def allocate(self,
                  covariance_matrix=None,
                  expected_asset_returns=None,
                  allocation_metric='equal_weighting',
-                 linkage='average',
+                 linkage='ward',
                  confidence_level=0.05,
                  optimal_num_clusters=None,
                  resample_by=None):
         # pylint: disable=too-many-arguments
         """
         Calculate asset allocations using the HCAA algorithm.
 
-        :param asset_names: (list) A list of strings containing the asset names
-        :param asset_prices: (pd.DataFrame) A dataframe of historical asset prices (daily close)
+        :param asset_names: (list) a list of strings containing the asset names
+        :param asset_prices: (pd.DataFrame) a dataframe of historical asset prices (daily close)
                                             indexed by date
-        :param asset_returns: (pd.DataFrame/numpy matrix) User supplied matrix of asset returns
-        :param covariance_matrix: (pd.DataFrame/numpy matrix) User supplied covariance matrix of asset returns
-        :param expected_asset_returns: (list) A list of mean asset returns (mu)
-        :param allocation_metric: (str) The metric used for calculating weight allocations. Supported strings - "equal_weighting",
+        :param asset_returns: (pd.DataFrame/numpy matrix) user supplied matrix of asset returns
```

Comment:
Why did you drop from starting a sentence with a capital? I think they need to be capital letters with full stops at the end of a sentence.

Reply:
Done